### PR TITLE
fix(all-pages): remove empty styles causing occasional build failure

### PR DESCRIPTION
**Pull request by shanberg**
I had been using empty Stylefy `sub-styles` as placeholders in the All Pages table, expecting to need them later. They weren't necessary, and leaving them in there was sloppy, but I didn't see any errors or breakage related to them at the time.

Then yesterday I was seeing incomplete style processing, and eventually narrowed it down to those empty sub-styles. I've removed them and added a note in the relevant location in the DevCards docs.

### CODE REVIEW

File: src/cljs/athens/devcards/all_pages.cljs [hunk 1]
```diff
@@ -77,10 +73,10 @@
                      db/dsdb)
         pages (pull-many db/dsdb '["*" {:block/children [:block/string] :limit 5}] @page-eids)]
     [:table (use-style table-style)
-     [:thead (use-sub-style table-style :thead)
+     [:thead
       [:tr
-       [:th (use-sub-style table-style :th-title) [:h5 "Title"]]
-       [:th (use-sub-style table-style :th-body) [:h5 "Body"]]
+       [:th  [:h5 "Title"]]
+       [:th  [:h5 "Body"]]
```
Comment (tangjeff0) [inline src/cljs/athens/devcards/all_pages.cljs:79]: extra space?

File: src/cljs/athens/devcards/styling_with_stylefy.cljs [hunk 2]
```diff
@@ -110,6 +110,8 @@
 
 (defcard-doc "
   Apply sub-styles with Stylefy's `(use-sub-style)` function.
+              
+  Don't create empty sub-styles, as this may silently break Stylefy processing.
```
Comment (tangjeff0) [inline src/cljs/athens/devcards/styling_with_stylefy.cljs:114]: worth making a PR to stylefy?

File: src/cljs/athens/devcards/styling_with_stylefy.cljs [hunk 3]
```diff
@@ -132,3 +134,20 @@
...
+  ```clojure
+  [:div (use-style cursor-trail-style) {:style {:left x :top y}}]
+  ```")
+
+
+(defcard-doc "
+  
+  ```clojure
+  [:div (use-style cursor-trail-style) {:style {:left x :top y}}]
+  ```")
```
Comment (tangjeff0) [inline src/cljs/athens/devcards/styling_with_stylefy.cljs:153]: duplicate

File: src/cljs/athens/devcards/styling_with_stylefy.cljs [hunk 4]
```diff
@@ -132,3 +134,20 @@
...
+
+
+(defcard-doc "
+  Avoid creating styles that will be frequently updated, because this forces Stylefy to create a new class for each update.
+  
+  In these cases, pass the style directly to the element to update it inline.
+  
+  ```clojure
+  [:div (use-style cursor-trail-style) {:style {:left x :top y}}]
+  ```")
```
Comment (tangjeff0) [inline src/cljs/athens/devcards/styling_with_stylefy.cljs:146]: Needs `merge`

```
(defn button-with-dynamic-position [text color x y]
  [:button (merge (use-style (custom-primary-button-style color))
                  {:style {:left x :top y}})
    text])
```
^ 
Also makes more sense to wrap in a function to emphasize parameterization. 

https://github.com/Jarzka/stylefy#combine--parametrise-styles

File: src/cljs/athens/devcards/styling_with_stylefy.cljs (hunk 2 above)
Comment (shanberg) [reply src/cljs/athens/devcards/styling_with_stylefy.cljs:114]: Not an area I can reasonably expect to make helpful changes, but I'll think about what to do.

**Pull request merged**